### ISSUE TITLE
mwan3 update: internet connectivity issue after update to version 2.11.16-6

### ISSUE THREAD

**Issue by gsanchietti** [NethServer/nethsecurity#985]
**Steps to reproduce**
1. Update machines using NethSecurity version 1.3.0 to mwan3 version 2.11.16-6.
2. Observe internet connectivity behavior for the machine and its clients.

**Expected behavior**
- Machines and their clients should maintain internet connectivity after updating to the new version of mwan3.

**Actual behavior**
- Machines remain accessible from the external network, but the machines themselves and their clients lose the ability to browse the internet.
- System logs indicate that the package `mwan3` overwrites the existing configuration file during the update:
  ```
  Dec 19 04:09:38 mam-firewall update-packages: Upgrading mwan3 on root from 2.11.8-1 to 2.11.16-6...
  Dec 19 04:09:38 mam-firewall update-packages:  * resolve_conffiles: Existing conffile /etc/mwan3.user is different from the conffile in the new package. The new conffile will be placed at /etc/mwan3.user-opkg.
  Dec 19 04:09:39 mam-firewall mwan3rtmon[2490]: failed to add 172.21.30.0/24 via 172.21.30.1 dev tun-nsplug to table 1 - error: RTNETLINK answers: Network unreachable
  ```

**Workaround**
Replace the configuration for `mwan3` and restart it:
```bash
echo <<EOF >/etc/config/mwan3
   config globals 'globals'
       option mmx_mask '0x3F00'
EOF
/etc/init.d/mwan3 restart
```

Or reset the mwan configuration as [documented](https://docs.nethsecurity.org/en/latest/multiwan.html#reset-configuration).

**Components**
NethSecurity version: 8-23.05.5-ns.1.3.0

**Comment by gsanchietti** [comment]
**Test case**

Check the issue is not reproducible

**Comment by gsanchietti** [comment]
Verified

**Comment by gsanchietti** [comment]
Released inside image 1.4.1
